consensus/ibft/ibft.go

```diff
@@ -751,12 +785,8 @@ func (i *Ibft) runAcceptState() { // start new round
 	}
 
 	if i.bridge != nil {
+		// TODO: Check threshold
```

Comment:
Leftover todo

Reply:
This is not leftover. We need to decide what threshold is? 2/3 validators? or full validators?